Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -147,6 +148,11 @@ namespace EMotionFX
         }
     }
 
+    void ColliderPropertyNotify::AfterPropertyModified([[maybe_unused]] AzToolsFramework::InstanceDataNode* pNode)
+    {
+        PhysicsSetupManipulatorRequestBus::Broadcast(&PhysicsSetupManipulatorRequests::OnUnderlyingPropertiesChanged);
```

Comment:
question: This might be fine for now but it could be useful to use Event and a more precise Id here to only notify the manipulator (or collider) that needs to change (instead of all of them). May not be a problem now but if there are many listeners it could get slower over time.

Reply:
Only active manipulators will be listening, and there shouldn't ever be many of them. The way it's set up at the moment there can only be one active at a time.

Reply:
Okay cool good to know, you can ignore this for now, it's just something I try and do if possible